samples/estimation/estimation-ising-2D.ipynb

```diff
@@ -0,0 +1,509 @@
+        "$$ \\text{Target rQOPS} = \\frac{\\text{Logical qubits}\\cdot\\text{Logical Depth}}{\\text{Target runtime}}$$"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "def get_target_rqops(results, labels):\n",
+        "\n",
+        "    target_runtime = 172800\n",
+        "    logical_qubits = []\n",
+        "    logical_depth = []\n",
+        "    target_rqops = []\n",
+        "    logical_error = []\n",
+        "\n",
+        "    for i in range(3): \n",
+        "        logical_qubits.append(results[i]['physicalCounts']['breakdown']['algorithmicLogicalQubits'])\n",
+        "        logical_depth.append(results[i]['physicalCountsFormatted']['logicalDepth'])\n",
+        "        logical_error.append(results[i]['physicalCountsFormatted']['requiredLogicalQubitErrorRate'])\n",
+        "        target_rqops.append(round(results[i]['physicalCounts']['breakdown']['algorithmicLogicalQubits'] * results[i]['physicalCounts']['breakdown']['logicalDepth'] / target_runtime))\n",
+        "\n",
+        "    data = pd.DataFrame()\n",
+        "    pd.options.display.float_format = '{:.2E}'.format\n",
+        "    data['Logical qubits'] = logical_qubits\n",
+        "    data['Logical depth'] = logical_depth\n",
+        "    data['Logical error'] = logical_error\n",
+        "    data['Target rQOPS'] = target_rqops\n",
+        "    data.index = labels\n",
+        "\n",
+        "    return data\n",
+        "\n",
+        "rQOPS_table = get_target_rqops(results, labels)\n",
+        "rQOPS_table\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Next steps\n",
+        "\n",
+        "Feel free to use this notebook as a starting point for your own experiments.  For\n",
+        "example, you can\n",
+        "\n",
+        "* explore how the results change considering other problem instances of the Heisenberg model\n",
+        "* explore space- and time-trade-offs by changing the value for\n",
+        "  `logical_depth_factor` or `max_t_factories`\n",
+        "* visualize these trade-offs with the space and time diagrams\n",
+        "* use other or customized qubit parameters"
+      ]
+    }
+  ],
+  "metadata": {
+    "kernel_info": {
+      "name": "python3"
+    },
+    "kernelspec": {
+      "display_name": "Python 3 (ipykernel)",
+      "language": "python",
+      "name": "python3"
+    },
+    "language_info": {
+      "codemirror_mode": {
+        "name": "ipython",
+        "version": 3
+      },
+      "file_extension": ".py",
+      "mimetype": "text/x-python",
+      "name": "python",
+      "nbconvert_exporter": "python",
+      "pygments_lexer": "ipython3",
+      "version": "3.11.7"
+    },
+    "microsoft": {
```

Comment:
This section is possibly not needed - I think it appears in notebooks that were downloaded from hosted notebooks, but now it's pointing to an archived repo. I deleted it from my notebooks

Reply:
Done. Removed the reference to hosted notebooks.